rama-dns/src/lib.rs

```diff
@@ -129,3 +90,7 @@ pub use in_memory::{DnsOverwrite, DomainNotMappedErr, InMemoryDns};
 mod deny_all;
 #[doc(inline)]
 pub use deny_all::{DenyAllDns, DnsDeniedError};
+
+pub mod chain;
+
+pub mod variant;
```

Comment:
```suggestion
mod variant;
```

AFAIK no reason to have this public. It's just about implementing something.